clr_loader/__init__.py

```diff
@@ -20,6 +22,8 @@ def get_mono(
 
     impl = Mono(
         domain=domain,
+        debug=debug,
+        jit_options=jit_options,
```

Comment:
Seeing the parameters are also positional, please append the new ones at the end of the existing list.

Reply:
Updated in `get_mono`, kept as is for `Mono.__init__` as in the later parameters are keyword only